src/srvctl/srvctl.cpp

```diff
@@ -0,0 +1,127 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2022, KNS Group LLC (YADRO).
+ */
+
+#include "srvctl.hpp"
+
+namespace srvctl
+{
+
+/** NOTE: This map was written by knowledges extracted from `bmcweb`
+ *        (https://github.com/openbmc/bmcweb) and `service-control-manager`
+ *        (https://github.com/openbmc/service-config-manager)
+ *
+ *        I dislike the such hard code, but this is a quick solution.
+ *        In further it should be generating at compile time or even better
+ *        retrieving via d-bus in run-time. But this requires a corresponding
+ *        implementation at service-manager side.
+ */
+ServiceDefinitions serviceDefinitions = {
+    {"SSH",
+     {
+         "SSH for BMC CLI shell",
```

Comment:
Yes. All this stuff is implemented inside `service-config-manager` and it works fine. Moreover,  specifically ssh-sessions destroy on disabling `SSH`, as I guess,  because of  your efforts in `obmc-yadro-session-manager`. ;)
This seems to me now like an over security, but I don't intend to change this.